omnibus/Gemfile.lock

```diff
@@ -132,7 +132,7 @@ GEM
       train-winrm (>= 0.2.5)
       uuidtools (>= 2.1.5, < 3.0)
       vault (~> 0.16)
-      win32-api (~> 1.5.3)
+      win32-api (~> 1.5)
```

Comment:
is this related to the bug? Should this be a separate PR?

Reply:
restoring that